x-pack/plugins/security_solution/public/management/cypress/support/create_and_enroll_endpoint_host_ci.ts

```diff
@@ -66,13 +68,13 @@ export const createAndEnrollEndpointHostCI = async ({
   hostname,
   version = kibanaPackageJson.version,
   useClosestVersionMatch = true,
-  isMkiEnvironment = false,
+  forceVersion = false,
+  isServerless = false,
```

Comment:
Same comment here. Don't need `isServerless`. you should just use the utility along with `esClient` or `kbnClient` to determine that.

Reply:
Changed.